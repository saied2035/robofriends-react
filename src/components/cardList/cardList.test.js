import {shallow} from 'enzyme'
import CardList from './cardList'

describe("render cardList",() => {
        it("no error case",() => {
            const props = {
                            robots: [
                                          {
                                            id: 1,
                                            name: 'saied',
                                            email: 'saiedgaffer@gmail.com'
                                           }
                                        ],
                            error: ""            
                        }
            
            expect(shallow(<CardList {...props} />)).toMatchSnapshot()
            
        })

        it("error case",() => {
            const props = {
                            mockRobots: [],
                            error: "error"            
                          }
            
            expect(shallow(<CardList {...props} />)).toMatchSnapshot()
            
        })            
})
