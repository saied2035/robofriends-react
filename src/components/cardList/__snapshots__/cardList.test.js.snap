// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render cardList error case 1`] = `
<div
  className=" f3 flex justify-center items-center"
  style={
    Object {
      "height": "70vh",
      "width": "100%",
    }
  }
>
  <h3>
    the robots not found for now something went wrong
  </h3>
</div>
`;

exports[`render cardList no error case 1`] = `
<div
  className="flex justify-center flex-wrap"
  style={
    Object {
      "height": "75vh",
      "overflowY": "auto",
    }
  }
>
  <Memo(Card)
    email="saiedgaffer@gmail.com"
    id={1}
    key="1"
    name="saied"
  />
</div>
`;
